docs/articles/documentation/guides/continuous-integration/circleci-and-lambdatest.md

```diff
@@ -0,0 +1,171 @@
+---
+layout: docs
+title: Run Tests on LambdaTest with CircleCI
+permalink: /documentation/guides/continuous-integration/circleci-and-lambdatest.html
+---
+# Run Tests on LambdaTest with CircleCI
+
+This topic describes how to integrate TestCafe into the [CircleCI](https://circleci.com/) build process and run tests in the [LambdaTest](https://www.lambdatest.com) cloud testing service.
+
+## Prerequisites
+
+In this tutorial, we will fetch tests from a GitHub repository: [testcafe-ci-demo](https://github.com/DevExpress-Examples/testcafe-ci-demo). Fork this repository before we start.
+
+> If you use a different version control system, make sure CircleCI supports it. To learn which systems it supports, refer to the [CircleCI Integrations](https://circleci.com/integrations/) page.
+
+## Step 1 - Install TestCafe and Create Tests
+
+Install TestCafe [locally](../basic-guides/install-testcafe.md#local-installation) in your project and [create tests](../../getting-started/README.md#creating-a-test).
+
+## Step 2 - Create a New CircleCI Project
+
+Log in to CircleCI using your GitHub account.
+
+If this is the first time you are using CircleCI, you will start from an empty workspace. Click the **Add Project** button to create your first project.
+
+![Click the Add Project Button](../../../images/circleci-lambdatest/add-project.png)
+
+You will see the list of GitHub projects associated with your account.
```

Comment:
```suggestion
to show a list of GitHub projects associated with your account.
```

Reply:
combine this with the previous sentence

Reply:
You mean this sentence?

> ![Click the Add Project Button](../../../images/circleci-lambdatest/add-project.png)

This is image's alt text, it won't be displayed.

Reply:
Ok, then change it to: 

```suggestion
This shows a list of GitHub projects associated with your account.
```